test/boost/view_schema_test.cc

```diff
@@ -3104,7 +3104,7 @@ SEASTAR_TEST_CASE(test_view_update_generating_writetime) {
         // The same scenario as before, but this time the second statement specifies TTL,
         // so updating the cell WILL generate a view update for each MV too.
         e.execute_cql("UPDATE t USING TIMESTAMP 7 SET g=40 WHERE k=1 AND c=1;").get();
-        e.execute_cql("UPDATE t USING TTL 10 AND TIMESTAMP 8 SET g=40 WHERE k=1 AND c=1;").get();
+        e.execute_cql("UPDATE t USING TTL 300 AND TIMESTAMP 8 SET g=40 WHERE k=1 AND c=1;").get();
```

Comment:
Maybe eventually() shouldn't have defaulted to 263 seconds (!), but as long as it does I agree the TTL (which we don't intend to delete the item, just to exist) should be higher. So good.